netlify.toml

```diff
@@ -505,6 +505,276 @@
   status = 301
   force = true
 
+[[redirects]]
+  from = "/guides/cloud/debugging/recorded-runs"
+  to = "/guides/cloud/recorded-runs"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "/api/commands"
+  to = "/api/table-of-contents"
+  status = 301
+  force = true
+
+[[redirects]]
```

Comment:
Not sure if there's a quick way to test this, but would the previous entry suffice without another including the trailing slash?